src/mlang/backend_compilers/bir_to_dgfip_c.ml

```diff
@@ -819,13 +819,6 @@ let generate_rovs_files (dgfip_flags : Dgfip_options.flags) (program : program)
 #define add_erreur(a,b,c) add_erreur(b,c)
 #endif
 
-#if defined(__STDC_VERSION__) && (__STDC_VERSION__ >= 199901L)
```

Comment:
Where is the replacement for those?

Reply:
_fmin/_fmax are replaced in the generated code by min/max. 
min/max are already defined as macros (equivalent to the _fmin/_fmax when C < 1999 was used) in the static const.h file, included by the function `generate_header` line 583.

Reply:
Ok they're here 

https://github.com/MLanguage/mlang/blob/510b5d457f2a6c42c729c70fdfae7a70e986a60e/examples/dgfip_c/const.h#L109-L110